parser/cellmata.g4

```diff
@@ -165,8 +161,9 @@ PAREN_START : '(' ;
 PAREN_END : ')' ;
 END : ';' ;
 
-OP_COMPARE : 'is' ;
-OP_NOT : 'not' ;
+OP_COMPARE : '==' ;
+OP_COMPARE_not : '!=' ;
```

Comment:
Whoops, seems like the token `OP_COMPARE_not`, note the lower case, does not match the applied usage at line 96. Please fix.

Reply:
Typo has been corrected in 8a667a6db564bfe5beceb4fe20afe51db317beab